art/metrics/metrics.py

```diff
@@ -32,17 +32,22 @@
 from tqdm.auto import tqdm
 
 from art.config import ART_NUMPY_DTYPE
+from art.attacks.attack import EvasionAttack
+from art.attacks.evasion.auto_attack import AutoAttack
 from art.attacks.evasion.fast_gradient import FastGradientMethod
 from art.attacks.evasion.hop_skip_jump import HopSkipJump
 from art.utils import random_sphere
 
 if TYPE_CHECKING:
-    from art.attacks.attack import EvasionAttack
     from art.utils import CLASSIFIER_TYPE, CLASSIFIER_LOSS_GRADIENTS_TYPE, CLASSIFIER_CLASS_LOSS_GRADIENTS_TYPE
 
 logger = logging.getLogger(__name__)
 
 SUPPORTED_METHODS: Dict[str, Dict[str, Any]] = {
+    "auto": {
+        "class": AutoAttack,
+        "params": {"eps_step": 0.1, "eps_max": 1.0},
```

Comment:
The default parameters most likely assume and input value range of [0, 1]. Should we raise a warning if we detect input values that are much larger than 1?

Reply:
Added the check.